odoo_module_migrate/migration.py

```diff
@@ -119,6 +135,13 @@ def _get_code_from_previous_branch(self, module_name, remote_name):
                 'target': target_version,
                 'module': module_name,
             }, path=self._directory_path)
+        if self._auto_squash:
+            logger.info("AutoSquash Commit")
+            _execute_shell(
```

Comment:
Not sure to understand that point.

this will work if you didn't install the lib. (if you just git clone and run locally your script ?)

why don't you call directly a function present in ``odoo_module_migrate/squasher.py`` instead of using shell ?